pytest_selenium/drivers/cloud.py

```diff
@@ -30,13 +30,14 @@ def config(self):
         config.read([name, os.path.join(os.path.expanduser('~'), name)])
         return config
 
-    def get_credential(self, key, env):
-        try:
-            value = self.config.get('credentials', key)
-        except (configparser.NoSectionError,
-                configparser.NoOptionError,
-                KeyError):
-            value = os.getenv(env)
-        if not value:
-            raise MissingCloudCredentialError(self.name, key, env)
-        return value
+    def get_credential(self, key, envs):
+        for env in envs:
```

Comment:
If `env` is a string, then we'll iterate over each character. Perhaps this isn't an issue as we're always passing a list, and we don't expect users to call this directly.